src/configureWorkspace/configureDotNetCore.ts

```diff
@@ -344,11 +417,16 @@ export async function scaffoldNetCore(context: ScaffolderContext): Promise<Scaff
     const dockerFileName = path.join(context.outputFolder ?? rootRelativeProjectDirectory, 'Dockerfile');
     const dockerIgnoreFileName = path.join(context.outputFolder ?? '', '.dockerignore');
 
-    const files: ScaffoldFile[] = [
+    const composeFiles = IsCompose ? generateComposeFiles(dockerFileName, context.platform, os, ports, workspaceRelativeProjectFileName) : [];
+
+    let files: ScaffoldFile[] = [
         { fileName: dockerFileName, contents: dockerFileContents, open: true },
         { fileName: dockerIgnoreFileName, contents: genCommonDockerIgnoreFile(context.platform) }
     ];
 
+    // TODO test if adding an empty array is no-op
```

Comment:
Empty arrays passed to `concat()` will no-op.  Undefined values (e.g. an `undefined` "array") passed to `concat()` will be result in the `undefined` value being added to the array.

Reply:
Thanks, I tested it, but forgot to remove the comment.